Strings/NPCs/NPCs.md

```diff
@@ -32,8 +86,34 @@
 
 | Language | Name |
 | :------- | :---------- |
-| Japanese | Translation needed |
+| Text ID | 911062 |
+|||
+| Japanese | 仙峯寺薙刀術士　峯治郎 <br> Senpou Glaiver - Houjirou |
 | Korean | Translation needed |
 | German | Meister der Glefe Houjiro |
 | Italian | Translation needed |
-| Spanish | Translation needed |
\ No newline at end of file
+| Spanish | Translation needed |
+
+# Bridgekeeper Shiramatsu
+
+| Language | Name |
+| :------- | :---------- |
+| Text ID | 911059 |
+|||
+| Japanese | 橋守　シラマツ |
+| Korean | Translation needed |
+| German | Translation needed |
+| Italian | Translation needed |
+| Spanish | Translation needed |
```

Comment:
Question : Should we put placeholders in other md files as well for Spanish language?

Reply:
@TotallyNotShinobi 
If possible please add the placeholders for other languages only in the files you are currently sending for the PR

Reply:
@Phan